testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -110,6 +110,7 @@
 end
 
 When(/^I wait until I see the name of "([^"]*)", refreshing the page$/) do |host|
+  raise 'Overview System page didn\'t load' unless has_content?('System Overview')
```

Comment:
do we want to just have the raise and not a solution to wait a bit more for this to be done?

Reply:
I will wait for the CapybaraTimeout ( has_XXX? wait by default ). So in PRV BV case, for 60 sec.